scripts/pre-commit.php

```diff
@@ -296,23 +287,6 @@ function check_opt($options)
     return false;
 }
 
-/**
- * Build a list of exclude arguments from an array
- *
- * @param string $exclude_string such as "--exclude"
- * @param array $excludes array of directories to exclude
- * @return string resulting string
- */
-function build_excludes($exclude_string, $excludes)
```

Comment:
You can't remove this, it's still used:

`$lint_exclude = build_excludes('--exclude ', $lint_excludes);`

Reply:
Whiopsie